ajax/libs/jqgrid/package.json

```diff
@@ -1,21 +1,48 @@
 {
   "name": "jqgrid",
   "filename": "js/jquery.jqGrid.min.js",
-  "version": "4.6.0",
+  "version": "5.0.1",
   "description": "jqGrid is an Ajax-enabled JavaScript control that provides solutions for representing and manipulating tabular data on the web.",
   "homepage": "http://www.trirand.com/blog/",
   "keywords": [
+    "jQuery",
+    "jqGrid",
+    "JavaScript",
     "grid",
-    "jquery"
+    "table",
+    "pivot",
+    "tree",
+    "subgrid",
+    "sorting",
+    "paging",
+    "editing",
+    "grouping",
+    "searching",
+    "ajax"
   ],
-  "license": "CC-BY-NC-3.0",
+  "license": "Creative Commons Attribution-Non-Commercial 3.0 License",
```

Comment:
As the comment https://github.com/cdnjs/cdnjs/pull/6007#issuecomment-318597375, We use `CC-BY-NC-3.0` which is the short identifier of `Creative Commons Attribution Non Commercial 3.0`.
https://spdx.org/licenses/CC-BY-NC-3.0.html
https://github.com/tonytomov/jqGrid/blob/master/bower.json#L12

Could you modify from `"license": "Creative Commons Attribution-Non-Commercial 3.0 License",` to `"license": "CC-BY-NC-3.0",`?
Thank you.